cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/BusbarSectionConversion.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.cgmes.conversion.elements;
+
+import com.powsybl.cgmes.conversion.Context;
+import com.powsybl.iidm.network.BusbarSection;
+import com.powsybl.iidm.network.BusbarSectionAdder;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class BusbarSectionConversion extends AbstractConductingEquipmentConversion {
+
+    public BusbarSectionConversion(PropertyBag bbs, Context context) {
+        super("BusbarSection", bbs, context);
+    }
+
+    @Override
+    public boolean valid() {
+        // We only can convert busbar sections if we are in node-breaker context
+        return context.nodeBreaker();
+    }
+
+    @Override
+    public void convert() {
+        BusbarSectionAdder bbsAdder = voltageLevel().getNodeBreakerView().newBusbarSection()
+                .setId(iidmId())
+                .setName(iidmName())
+                .setEnsureIdUnicity(false);
+        bbsAdder.setNode(iidmNode());
+        BusbarSection bbs = bbsAdder.add();
+        convertedTerminals(bbs.getTerminal());
```

Comment:
you will need to check either here or in ```convertedTerminals``` that ```p``` and ```q``` are not set for ```BusbarSection``` and other ```Connectable``` where it is forbidden

Reply:
Added the condition in convertedTerminals. We do not try to set p, q for terminals which do not allow it: currently if the connectable type is a bus bar section.